### DISCUSSION TITLE
Select Customer in POS

### DISCUSSION THREAD

**Discussion by bonisaputra** [Blair2004/NexoPOS#794] [Q&A]
hello, i have a problem and question
1. how can i select customer from POS page ? its only shown default customer or create new customer in customer tab button

2. Print button in Order PopUp in POS can't be click, no action at all. how to handle it

i am using 4.7.8 version

**Comment by Blair2004** [comment]
Hi, for the customer https://www.youtube.com/watch?v=ViMvX00KpJE
We'll investigate on the print button.

**Reply by bonisaputra** [reply]
> Hi, for the customer https://www.youtube.com/watch?v=ViMvX00KpJE We'll investigate on the print button.

for print button problem, solving with by set in setting POS tab Print and select Printing Enabled For => All Orders

now the problem is, cant remove Default Customer Account and Default Customer Group in setting Page, i have to set at least one, or delete current selected customer so in POS page i can select customer again.

**Reply by Blair2004** [reply]
A customer group and default customer shouldn't be deleted once it's defined. Make sure to at least provide one.

**Reply by bonisaputra** [reply]
if i provide it, i cant select another customer account. only showing the default customer. even i reset transaction

**Reply by Blair2004** [reply]
You can't select another customer account from here ?

**Reply by bonisaputra** [reply]
change font color< H2 > tag in customer account pop up too, its white

**Reply by Blair2004** [reply]
For the issue you're facing, consider creating a dedicated thread on the issues panel. https://github.com/Blair2004/NexoPOS-4x/issues
Make sure to see #501 to understand how to create an issue.

**Reply by bonisaputra** [reply]
> You can't select another customer account from here ?

yes, if Default Customer Account chosen in setting

**Reply by Blair2004** [reply]
I'll invite you for all the concerns you're facing to open a dedicated issue on the issues panel. We'll handle each separately.
Thanks

**Reply by bonisaputra** [reply]
OK thenks, i've publish the issue
https://github.com/Blair2004/NexoPOS-4x/issues/795
